Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertGeometry.cs

```diff
@@ -1259,7 +1259,12 @@ public Brep BrepToSpeckle(Solid solid, Document d, string units = null)
             // Update trim indices with new item.
             // TODO: Make this better.
             var trimIndices = sEdge.TrimIndices.ToList();
-            trimIndices.Append(sTrimIndex); //TODO Append is a pure function and the return is unused
+
+            // suppressing the warning even though it is completely valid.
+            // reason we are just suppressing is because this entire class is broken anyways
+#pragma warning disable CA1806 // Do not ignore method results
+            trimIndices.Append(sTrimIndex);
```

Comment:
This might be true, but as this line isn't doing anything surely we need either a Add or AddRange() here?